python/pyspark/sql/functions/builtin.py

```diff
@@ -21314,6 +21314,109 @@ def make_timestamp(
         )
 
 
+@_try_remote_functions
+def try_make_timestamp(
```

Comment:
@HyukjinKwon @zhengruifeng Could you look at the PR, please, especially at the python-related changes.